### PR TITLE
[RTM] More documentation fixes and cleanup

### PR DESCRIPTION
**Pull request by chrisfilo**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -35,9 +41,10 @@ For security reasons, many HPCs (e.g., TACC) do not allow Docker containers, but
 In this case, start with a machine (e.g., your personal computer) with Docker installed.
 Use `docker2singularity <https://github.com/singularityware/docker2singularity>`_ to create a singularity image. You will need an active internet connection and some time. ::
 
-    $ docker run -v /var/run/docker.sock:/var/run/docker.sock \
-        -v D:\host\path\where\to\ouptut\singularity\image:/output \
-        --privileged -t --rm singularityware/docker2singularity \
+    $ docker run --privileged -t --rm \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v D:\host\path\where\to\output\singularity\image:/output \
```
Comment (effigies) [inline docs/installation.rst]: Does docker2singularity use a `D:\`-style filesystem for the host, or is this assuming a Windows user?

File: docs/installation.rst [hunk 2]
```diff
@@ -47,14 +54,17 @@ Transfer the resulting Singularity image to the HPC, for example, using ``scp``.
...
 
 For example: ::
 
     $ singularity run ~/poldracklab_fmriprep_latest-2016-12-04-5b74ad9a4c4d.img \
+        /work/04168/asdf/lonestar/ $WORK/lonestar/output \
+        participant \
         --participant_label sub-387 --nthreads 1 -w $WORK/lonestar/work \
-        --ants-nthreads 16 --skull--strip-ants /work/04168/asdf/lonestar/ \
-        $WORK/lonestar/output participant
+        --ants-nthreads 16 --skull--strip-ants
```
Comment (effigies) [inline docs/installation.rst]: `--skull-strip-ants`?

File: docs/installation.rst (hunk 1 above)
Comment (chrisfilo) [inline docs/installation.rst]: This was assuming using "Docker for Windows".

Comment (oesteban) [inline docs/installation.rst]: But the previous line says ```-v /var/run/docker.sock:/var/run/docker.sock``` is that correct?

**Comment by chrisfilo** [comment]
Counterintuitive, but this is how Docker for Windows works. This socket is
special.

File: docs/installation.rst [hunk 3]
```diff
@@ -47,14 +54,17 @@ Transfer the resulting Singularity image to the HPC, for example, using ``scp``.
...
+        participant \
+        --participant_label label
 
 For example: ::
 
     $ singularity run ~/poldracklab_fmriprep_latest-2016-12-04-5b74ad9a4c4d.img \
-        --participant_label sub-387 --nthreads 1 -w $WORK/lonestar/work \
-        --ants-nthreads 16 --skull--strip-ants /work/04168/asdf/lonestar/ \
-        $WORK/lonestar/output participant
+        /work/04168/asdf/lonestar/ $WORK/lonestar/output \
```
Comment (oesteban) [inline docs/installation.rst]: Do we want these particular paths tightly linked to TACC clusters?

File: docs/installation.rst [hunk 4]
```diff
@@ -47,14 +54,17 @@ Transfer the resulting Singularity image to the HPC, for example, using ``scp``.
...
+        participant \
+        --participant_label sub-387 --nthreads 16 -w $WORK/lonestar/work \
```
Comment (oesteban) [inline docs/installation.rst]: Should we not make the jump from ```--nthreads``` to ```--nprocs``` ASAP?

**Comment by chrisfilo** [comment]
What's the disadvantage? It's just an example.

**Comment by chrisfilo** [comment]
nprocs?

**Comment by oesteban** [comment]
Yes, I think ```--nthreads``` is a bit misleading. We had this discussion about MRIQC if I recall. Probably ```--nprocs``` or ```--n_procs``` is more appropriate, DYT?

**Comment by effigies** [comment]
LGTM, assuming agreement on `--nthreads`/`--nprocs` is reached. (I don't have an opinion here, is there a pointer to existing arguments?)

**Comment by chrisfilo** [comment]
Let's leave changes to the command line arguments to a separate PR.

**Pull request merged**